src/http-spec.ts

```diff
@@ -24,6 +24,12 @@ export interface IHttpService extends INode {
     url?: string;
     identifier?: string;
   };
+  logo?: {
+    url: string;
+    altText: string;
```

Comment:
This is optional then, according to their spec.

Reply:
@P0lip I changed the required props. Both `altText` and and `href` will have default values